<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="all_checkBox">
     <property name="text">
      <string>All</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="Line" name="poleVector_line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="PV_arm_label">
     <property name="text">
      <string>Arm</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="armLf_checkBox">
     <property name="text">
      <string>Lf</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="armRt_checkBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Rt</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="PV_leg_label">
     <property name="text">
      <string>Leg</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="legLf_checkBox">
     <property name="text">
      <string>Lf</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="legRt_checkBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Rt</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="Line" name="poleVector_line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="PV_controlOffset_Label">
     <property name="text">
      <string>Control Offset</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="controlOffset_doubleSpinBox">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="PV_sizeOfControl_label">
     <property name="text">
      <string>Size of Control</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QDoubleSpinBox" name="sizeOfControl_doubleSpinBox">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-100000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="Line" name="poleVector_line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="PV_CustomPoleVector_label">
     <property name="text">
      <string>Custom Pole Vector</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="PV_customSysName_label">
     <property name="text">
      <string>System Name</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QLineEdit" name="customSysName_lineEdit"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="PV_startJoint_label">
     <property name="text">
      <string>Start Joint</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="2">
    <widget class="QLineEdit" name="startJoint_lineEdit"/>
   </item>
   <item row="10" column="3">
    <widget class="QPushButton" name="startJoint_pushButton">
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="PV_endJoint_label">
     <property name="text">
      <string>End Joint</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="2">
    <widget class="QLineEdit" name="endJoint_lineEdit"/>
   </item>
   <item row="11" column="3">
    <widget class="QPushButton" name="endJoint_pushButton">
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="create_pushButton">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="delete_pushButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="reset_pushButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
